els-front/src/components/UpdateProfile.js

```diff
@@ -0,0 +1,114 @@
+import React,{ useRef, useState } from "react";
+import { Form, Field } from 'react-final-form'
+import { useNavigate } from "react-router-dom";
+import api from '../plugins/axios'
+import { successNotify } from "../plugins/toast";
+
+const UpdateProfile = ({user}) => {
+    const navigate = useNavigate();
+    const [image, setImage] = useState(user.avatar);
+    const inputAvatar = useRef(null);
+
+    const handleUpdate = async(formValues) => {
+        const response = await api.post(`user/update-credentials`, {...formValues,avatar : image})
+
+        if(response) {
+            successNotify('Updated profile successfully')
+            navigate(0)
+        }
```

Comment:
You need to check the response. In this case Wether the result success or fails, the notif will still be successful.

Reply:
It is in the axios interceptors sir